eucaconsole.init

```diff
@@ -60,9 +70,16 @@ start() {
     fi    
 
     if [ "x$NGINX_FLAGS" != xNO ]; then
-	echo -n "Starting eucaconsole nginx: "
+        echo -n "Starting eucaconsole nginx: "
         daemon /usr/sbin/nginx -c /etc/eucaconsole/nginx.conf $NGINX_FLAGS
-	[ $? -eq 0 ] && touch /var/lock/subsys/eucaconsole-nginx
+        [ $? -eq 0 ] && touch /var/lock/subsys/eucaconsole-nginx
+        echo
+    fi
+
+    if [ "x$MEMCACHED_FLAGS" != xNO ]; then
+        echo -n "Starting eucaconsole memcached: "
+        daemon --user=$CONSOLEUSER --pidfile ${MEMCACHEDPIDFILE} memcached -d -p $PORT -u $CONSOLEUSER  -m $MEMCACHED_CACHESIZE -c $MEMCACHED_MAXCONN -P ${MEMCACHEDPIDFILE} $MEMCACHED_FLAGS
```

Comment:
Okay.  In that case just fix the date in the changelog entry and this should be good to go.  I'll merge it tomorrow.

Thanks for the patch!